appcues/src/main/java/com/appcues/ui/utils/ActivityExt.kt

```diff
@@ -25,8 +25,16 @@ internal fun Activity.getParentView(): ViewGroup {
     // to find the top most decorView like below.
 
     val decorView = if (VERSION.SDK_INT >= VERSION_CODES.Q) {
-        // this is the preferred method on API 29+ with the new WindowInspector function
-        WindowInspector.getGlobalWindowViews().last()
+        try {
+            // this is the preferred method on API 29+ with the new WindowInspector function
+            // in case of multiple views, get the one that is hosting android.R.id.content
+            // we get the last one because sometimes stacking activities might be listed in this method,
+            // and we always want the one that is on top
+            WindowInspector.getGlobalWindowViews().last { it.findViewById<View?>(android.R.id.content) != null }
```

Comment:
in the comment starting on line 22 it says "However, if something like a dialog modal has been shown, this can add another window with another decorView on top of the Activity. If we want to support showing content above that layer, we need to find the top most decorView like below."

Were you able to test that scenario of presentation on top of a dialog modal as well? would that result in two views with `android.R.id.content`? Curious how that works here

Reply:
so yes for the dialog case it seems to work, just noting we found an issue with AlertDialog that when we add a view to that it seem to break the AlertDialog layout, but Dialog on RN still works